WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -787,26 +803,21 @@ class AbstractPostListViewController: UIViewController,
 
     // MARK: - Ghost cells
 
-    func startGhost() {
-        guard let ghostOptions = ghostOptions, emptyResults else {
+    final func startGhost() {
+        guard ghostingEnabled, emptyResults else {
             return
         }
 
-        let style = GhostStyle(beatDuration: GhostStyle.Defaults.beatDuration,
-                               beatStartColor: .placeholderElement,
-                               beatEndColor: .placeholderElementFaded)
-        tableView.displayGhostContent(options: ghostOptions, style: style)
-        tableView.isScrollEnabled = false
+        ghostableTableView.isHidden = false
         noResultsViewController.view.isHidden = true
     }
 
-    func stopGhost() {
-        tableView.removeGhostContent()
-        tableView.isScrollEnabled = true
+    final func stopGhost() {
+        ghostableTableView.isHidden = true
```

Comment:
I'm not that familiar with the ghosting functionality, but should we also be removing the ghost content when the ghost tableview is hidden? Does the animation continue (albeit hidden) if we don't?

Reply:
Very good question.

I did a quick research and wasn't able to find that answer.

So, I'm assuming that the animation would continue. In 2935c7c83b3f26bbf38db1e1c4189ae01337bed5 I start/stop the animation to avoid that.